models/comment.js

```diff
@@ -30,12 +31,28 @@ const commentSchema = new Schema(
       type: Schema.Types.ObjectId,
       ref: COOPERATION,
       required: [true, FIELD_CANNOT_BE_EMPTY('cooperation')]
-    },
+    }
   },
   {
     timestamps: true,
     versionKey: false
   }
 )
 
+commentSchema.post('save', async function () {
+  const cooperation = await cooperationService.getCooperationById(this.cooperation)
+
+  const user =
+    this.author.toString() === cooperation.initiator.toString() ? cooperation.receiver : cooperation.initiator
+  const notificationData = {
+    user,
+    userRole: user === cooperation.initiator ? cooperation.initiatorRole : cooperation.receiverRole,
+    type: 'new',

```

Comment:
maybe it would be good to create some enum object for these types

Reply:
done